Present.md

```diff
@@ -1,9 +1,11 @@
-In this section I will talk about where Faker is at currently in terms of its development. Its important to keep in consideration the past of Faker, in all its
-forms. 
+This section will focus on where Faker is currently developing. 
 
-Faker PHP was sunsetted due to design limitations. The creator of PHP faker, noted that the project was relatively large, sitting at 3 mb, and was used for 
-automated testing, which meant it was being downloaded quite frequently. To be specific, the creator estimated that the project was downloaded around 121 million
-times around the time of the sunset announcement. 121 M x 3 = 363 Million MB. 
```

Comment:
times around the time of the sunset announcement, meaning 363 million MB was downloaded.